### DISCUSSION TITLE
Is there a way to enable error squiggles?

### DISCUSSION THREAD

**Discussion by Android789515** [helix-editor/helix#9654] [Q&A] [answered]
The last time I had Helix installed I could swear when I had errors it would underline them with a red squiggle.

**Comment by kirawi** [comment, marked as answer]
It depends on your theme, [your terminal](https://github.com/helix-editor/helix/wiki/Terminal-Support), and whether you're using something like tmux or SSHing (where the terminfo is not picked up).

**Comment by chtenb** [comment]
You may also have to enable it in the config

undercurl = true
